Tests/jabTests/jabTests.swift

```diff
@@ -30,26 +30,30 @@ import XCTest
 //    Key -> Car is a one to one relationship
 //
 
-struct Car: Codable, Equatable {
+struct Car: Codable, Equatable, JSONAPIIdentifiable {
+    var identifier: String
     let color: String
```

Comment:
Bilo bi kul da ima test za Realm Object sa tipa List nečim, sam kaj to povlači da moraš imat realm ko dependency, al razmišljam tipa napravit novi unit testing target i stavit realm ko dependecy samo tamo? Ak je too much, sam lokalno provjerit realm object parsing.

Reply:
ne zelim imat testove sa 600+ MB dependencija...

Reply:
Pa zato bi odvojio u novi test target, to mi je onda nekak ok, ne?

Reply:
testiranje dekodiranja bilo kakvih custom objekata generalno nema smisla. decodeat ce se ako JSONDecoder moze proc. ovo je higher level i samo transformira JSON.